### DISCUSSION TITLE
DateSelector Block sometimes fails to pop up the date selector

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1220] [Questions and Help] [answered]
`DateSelector Block` sometimes fails to pop up the date selector

**Comment by Gervwyk** [comment, marked as answer]
Thanks, @xiaoxiji1234 for bringing this to our attention. We haven't seen it in our apps. Please do share if you find a reproducible config so we can debug and patch the issue. 
As far as the component itself goes, it is [Antd's DatePicker](https://ant.design/components/date-picker/) component, and it is very widely used and tested, we only put a thin wrapper / react component in front of it to map the properties to the cate component. So this an issue like this is very likely to also surface in the antd community. It could be that there has been a patch for the such an issue on the antd version that is shipped with v3 currently, I'll look into that. 
Also for v4 we'll be releasing with the most up to date version of antd, so would like to see if it still happens in v4.

**Reply by xiaoxiji1234** [reply]
I got it. Thank you!
